src/community/index.html

```diff
@@ -6,121 +6,128 @@
 
 <section class="community__intro row text-center">
     <div class="col-12">
-        <h1>Welcome to the Flutter community</h1>
+        <h1>Chào mừng đến với cộng đồng Flutter</h1>
         <p class="community__intro__description">
-            We're open source and would love to hear from you!
-            Below you'll find ways to get involved in the Flutter
-            community as well as links to
-            resources that can help answer your questions.
-            If you're in China, read
-            <a href="/community/china">Using Flutter in China</a>.
+            Chúng tôi là dự án mã nguồn mở và rất muốn nghe ý kiến từ bạn
```

Comment:
missing "!" or "."

Reply:
Sorry, but which ones are format-related changes? :( Besides that shall I translate all the descriptions under certain links?

Reply:
see https://github.com/truongsinh/website/pull/34#issuecomment-623561962 for example of format-related changes. For this particular request, I mean it's missing "!" or "." and the end of the sentence.